recipe/meta.yaml

```diff
@@ -11,36 +11,33 @@ source:
 
 build:
   number: 0
-  skip: True  # [py<37 or not linux]
+  skip: True  # [py<37 or not unix]
   script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv
 
 requirements:
   host:
     - pip
     - python
   run:
-    - aiofiles >=0.5.0,<0.6.0
+    - aiofiles >=0.7.0,<0.8.0
     - ansimarkup >=1.0.0
     - colorama >=0.4,<0.5
-    - click >=7.0
```

Comment:
Ah, got it. So we don't have `click` in b2, but this recipe is building b1. Just need to bump the version & hash up in this file @oliver-sanders . Then we won't need the bot autogenerated PR for the next release?

Reply:
Sure, so long as it is safe for us to do that manually.

Reply:
It should be fine. At least for the very first version we had to do that manually. If we release it manually, it could be that the bot immediately finds something missing (e.g. it could be build for py 3.10, or some new pypy version, etc). In that case the bot will just create a new PR with `buildnumber += 1`.